test/FunctionalTests/Tests/DataServices/UnitTests/DesignT4UnitTests/CodeGenerationContextTest.cs

```diff
@@ -294,5 +294,69 @@ public void GetEdmxStringFromMetadataPathShouldThrowExceptionIfInputIsInvalidUri
                 Assert.AreEqual(ex.Message, "Only file, http, https schemes are supported for paths to metadata source locations.");
             }
         }
+
+        [TestMethod]
+        public void SetOneCustomHeaderCorrectlyShouldNotThrowException()
+        {
+            var codegen = new ODataT4CodeGenerator();
+            string customHeaderString = @"Authorization:Bearer bearer-token";

```

Comment:
Would be better to be consistent in either preferring ```var``` or ```type```.
The style in this file seems to prefer ```type``` over ```var``` would be much better to follow that.

Reply:
I have change all to type